workflow.txt

```diff
@@ -1,10 +1,10 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
+git commit -m ""
 git checkout master
+git merge my-solution
```

Comment:
Very close!

Unless we've made more edits to our files we don't need to `add` and `commit` a second time, in fact the exercise wants us to keep our changes in the `my-solution` so we shouldn't be changing master this way

Instead, because other developers may have been pushing changes while we were working in our branch, the next step (before `git merge my-solution`) is to pull all those remote changes
`git pull upstream master` 
now we can merge and finally push our branch